Python/pywarpx/__init__.py

```diff
@@ -17,3 +17,5 @@
 from .PSATD import psatd
 from .Lasers import lasers
 from .Diagnostics import diagnostics
+
+from ._libwarpx import *
```

Comment:
One of the few use cases where `*` is totally valid, so we silence diagnostics here for all quality assurance tools:
```suggestion
from ._libwarpx import *  # noqa
```